### ISSUE TITLE
N-MNIST data gives error downloading

### ISSUE THREAD

**Issue by AArriandiaga** [neuromorphs/tonic#177]
```python
import tonic

dataset = tonic.datasets.NMNIST(save_to='./data', train=True)
events, target = dataset[0]
print(events)
```

gives :
> HTTP Error 404: Not Found

It seems that the Dropbox link is down. I also tried to use the OneDrive link but it gives the following error:

> File not found or corrupted.

**Comment by biphasic** [comment]
thanks for reporting that one. Will look into that

**Comment by biphasic** [comment]
the dataset owner (Garrick Orchard) is moving the dataset to a new location due to demand. Should hopefully be the case soon

**Comment by biphasic** [comment]
this is now fixed in 7070e823a83c68d9fc0bb725770cb9b881507ee8
I basically changed to a different data host
